src/Data/Opportunity/OpportunityDataClass.ts

```diff
@@ -6,13 +6,689 @@ import { pisaOpportunityDataClass } from './Pisa/pisaOpportunityDataClass'
+            { value: 'PSA_SAP_MAI_OPP_POT_LEA', title: 'Chance in Bewertung' },
+            { value: 'PSA_SAP_MAI_ORD_QUA', title: 'Zum Auftrag qualifiziert' },
+            {
+              value: 'PSA_SAP_MAI_QUO_ORD_PRP_ZUR2',
+              title: 'Chance zurückgestellt',
+            },
+            { value: 'PSA_SAP_MAI_QUO_QUA', title: 'Zum Angebot qualifiziert' },
+            { value: 'PSA_SAP_MAI_QUO_QUO_GEN', title: 'Chance in Erfassung' },
+          ],
+        }
+      : {
+          title: 'Main status',
+          values: [
+            { value: 'PSA_SAP_MAI_OPP_LST', title: 'Lost' },
+            { value: 'PSA_SAP_MAI_OPP_OPP_PRG', title: 'Uninteresting' },
+            { value: 'PSA_SAP_MAI_OPP_POT_LEA', title: 'Evaluation' },
+            { value: 'PSA_SAP_MAI_ORD_QUA', title: 'Qualified for order' },
+            { value: 'PSA_SAP_MAI_QUO_ORD_PRP_ZUR2', title: 'Deferred' },
+            { value: 'PSA_SAP_MAI_QUO_QUA', title: 'Qualified for quote' },
+            { value: 'PSA_SAP_MAI_QUO_QUO_GEN', title: 'Quote generation' },
+          ],
+        },
+  ] as const
+
+  const status = [
+    'enum',
+    lang === 'de'
+      ? {
+          title: 'Status',
+          values: [
+            { value: 'PSA_PRO_OPP_LST', title: 'Anfrage verloren' },
+            { value: 'PSA_PRO_OPP_TQU', title: 'zum Angebot qualifiziert' },
+            { value: 'PSA_PRO_OPP_TOR', title: 'zum Auftrag qualifiziert' },
+            {
+              value: 'PSA_PRO_OPP_PCR',
+              title: 'Anfrage pending (Gründe beim Kunden)',
+            },
+            { value: 'PSA_PRO_OPP_PSI_3', title: 'In ERP Teillieferung' },
+            { value: 'PSA_PRO_OPP_PSI_0', title: 'In ERP erfasst' },
+            {
+              value: 'PSA_PRO_OPP_PSI_1',
+              title: 'In ERP terminiert/abgesichert',
+            },
+            { value: 'PSA_PRO_OPP_PSI_2', title: 'In ERP mit Rechnung' },
+            {
+              value: 'PSA_PRO_OPP_PSI_4',
+              title: 'In ERP Teillieferung/Teilrechnung',
+            },
+            {
+              value: 'PSA_PRO_OPP_PSI_5',
+              title: 'In ERP Teillieferung/Vollrechnung',
+            },
+            { value: 'PSA_PRO_OPP_PSI_6', title: 'In ERP Lieferung' },
+            {
+              value: 'PSA_PRO_OPP_PSI_7',
+              title: 'In ERP Lieferung/Teilrechnung',
+            },
+            { value: 'PSA_PRO_OPP_PSI_8', title: 'In ERP storniert' },
+            { value: 'PSA_PRO_OPP_PSI_9', title: 'In ERP erledigt' },
+            { value: 'PSA_PRO_OPP_LIN', title: 'Lead mit Interesse' },
+            { value: 'PSA_PRO_OPP_KPI', title: 'kennt Produkt, hat Interesse' },
+            {
+              value: 'PSA_PRO_OPP_BPS',
+              title: 'hat Budget, Projekt startet in 6 Monaten',
+            },
+            {
+              value: 'PSA_PRO_OPP_DIS',
+              title: 'Anfrage uninteressant, kein Potential',
```

Comment:
* [ ] FYI I need a refresher for where/why we use opportunities in the portal. So far we only had Keyword and Description. The value here, for example, looks like internal data hat may be inappropriate in a portal context.

Reply:
=> moved discussion to slack.